### PR TITLE
doc(README.md): modifying the readme to reflect beta status

### PR DESCRIPTION
**Pull request by arschles**
cc/ @smothiki @mboersma @slack 

Ref https://github.com/deis/deis/issues/4962

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,95 +1,70 @@
...
-make deploy kube-service
-```
+## Dogfooding
+
+Please follow the instructions on the [official Deis docs](http://docs-v2.readthedocs.org/en/latest/installing-workflow/installing-deis-workflow/) to install and configure your Deis cluster and all related tools, and deploy and configure an app on Deis.
 
 ## License
 
-Copyright 2013, 2014, 2015 Engine Yard, Inc.
+Copyright 2013, 2014, 2015,2016 Engine Yard, Inc.
```
Comment (bacongobbler) [inline README.md]: missing an extra space for consistency

Comment (arschles) [inline README.md]: thanks! fixed in bd996039c916d01790270be7357a6586361652f7

File: README.md [hunk 2]
```diff
@@ -1,95 +1,70 @@
...
+## Beta Status
 
-Deis Minio v2 is changing quickly. Your feedback and participation are more than welcome, but be
-aware that this project is considered a work in progress.
+This Deis component is currently in beta status, and we welcome your input! If you have feedback, please submit an [issue][issues]. If you'd like to participate in development, please read the "Development" section below and submit a [pull request][prs].
 
 # About
 
-This package provides a [Minio](http://minio.io) S3-compatible object storage system on Kubernetes. It can be used as a [Deis](https://deis.com/) component to provide object storage for various other components, but it is flexible enough to be run anywhere else.
+The Deis minio component provides a [Minio](http://minio.io) server that can be run on Kubernetes. It's intended for use within the [Deis v2 platform](http://docs-v2.readthedocs.org/en/latest/) as an object storage server, but it's flexible enough to be run as a standalone pod on any Kubernetes cluster.
```
Comment (smothiki) [inline README.md]: Minio provides a standalone  S3 based object storage server

**Comment by smothiki** [comment]
ping @kmala 
Everything look fine other than my above comment . Leaving LGTMs to doc gurus again

File: README.md [hunk 3]
```diff
@@ -1,95 +1,70 @@
...
 
-We provide it as a Docker container, and also provide the following manifests to run it inside [Kubernetes](http://kubernetes.io/):
+Currently, we aren't providing this component with any kind of persistent storage, but it may work with [persistent volumes](http://kubernetes.io/docs/user-guide/volumes/).
```
Comment (smothiki) [inline README.md:26]: Before this it's better we mention. Minio backs deis-registry and builder

Comment (smothiki) [inline README.md:26]: Also database ??

File: README.md (hunk 2 above)
Comment (arschles) [inline README.md]: good call, fixed in c33e30a9a3a220273d441d85b1e9b8d6e4c25e50

File: README.md (hunk 3 above)
Comment (arschles) [inline README.md:26]: Fixed in 7b70b29741053d5e0812dbfab6a30af3f302f417

**Comment by arschles** [comment]
Thanks @smothiki

File: README.md [hunk 4]
```diff
@@ -1,95 +1,76 @@
...
+[![Build Status](https://travis-ci.org/deis/minio.svg?branch=master)](https://travis-ci.org/deis/minio)
+[![Go Report Card](http://goreportcard.com/badge/deis/minio)](http://goreportcard.com/report/deis/minio)
+[![Docker Repository on Quay](https://quay.io/repository/deisci/minio/status "Docker Repository on Quay")](https://quay.io/repository/deisci/minio)
 
-Deis (pronounced DAY-iss) is an open source PaaS that makes it easy to deploy and manage
-applications on your own servers. Deis builds on [Kubernetes](http://kubernetes.io/) to provide
-a lightweight, [Heroku-inspired](http://heroku.com) workflow.
+Deis (pronounced DAY-iss) is an open source PaaS that makes it easy to deploy and manage applications on your own servers. Deis builds on [Kubernetes](http://kubernetes.io/) to provide
+a lightweight, easy and secure way to deploy your code to production.
```
Comment (krancour) [inline README.md]: I feel like this section needs to be reworded to accurately reflect that Deis _Workflow_ is the PaaS.

Here's how it has been worded in docs-v2:

> Deis (pronounced DAY-iss) Workflow is an open source Platform as a Service (PaaS) that adds a developer-friendly layer to any Kubernetes cluster, making it easy to deploy and manage applications on your own servers.

https://github.com/deis/docs-v2/blob/master/src/index.md

Comment (arschles) [inline README.md]: @krancour thanks for that. fixed in e6633a6ff8618a7be834e0f1808e680fa29c9878 and noted in deis/builder#259 to be fixed, as that README has already been refactored and merged

**Pull request merged**